packages/stark-build/config/webpack.config.js

```diff
@@ -0,0 +1,282 @@
+"use strict";
+
+const helpers = require("./helpers");
+const buildUtils = require("./build-utils");
+const fs = require("fs");
+
+/**
+ * Webpack Plugins
+ */
+const DefinePlugin = require("webpack/lib/DefinePlugin");
+const MomentLocalesPlugin = require('moment-locales-webpack-plugin');
```

Comment:
@SuperITMan I think you can close this issue now: https://github.com/NationalBankBelgium/stark/issues/85

Good job! 👍

Reply:
Indeed! Thanks for the remark 👍